lib/data-to-table.au3

```diff
@@ -141,34 +142,34 @@ Func __SetConstants()
-            ['EdgeTopLeft',         '',    '┌',   '┌',   '┌',   '╔',   '╔',   '╔',   '╔'], _
-            ['EdgeTopRight',        '',    '┐',   '┐',   '┐',   '╗',   '╗',   '╗',   '╗'], _
-            ['EdgeBottomLeft',      '',    '└',   '└',   '└',   '╚',   '╚',   '╚',   '╚'], _
-            ['EdgeBottomRight',     '',    '┘',   '┘',   '┘',   '╝',   '╝',   '╝',   '╝'], _
+            ['',                     1,     2,     3,     4,     5,     6,     7,     8,     9 ], _
+            ['EdgeTopLeft',         '',    '┌',   '┌',   '┌',   '╔',   '╔',   '╔',   '╔',   '┌'], _
+            ['EdgeTopRight',        '',    '┐',   '┐',   '┐',   '╗',   '╗',   '╗',   '╗',   '┐'], _
+            ['EdgeBottomLeft',      '',    '└',   '└',   '└',   '╚',   '╚',   '╚',   '╚',   '└'], _
+            ['EdgeBottomRight',     '',    '┘',   '┘',   '┘',   '╝',   '╝',   '╝',   '╝',   '┘'], _
             _
-            ['TopCross',            '',    '┬',   '┬',   '┬',   '┬',   '╤',   '╦',   '╤'], _
-            ['HeaderCross',         '',    '┼',   '┼',   '┼',   '┼',   '┼',   '╬',   '┼'], _
-            ['BottomCross',         '',    '┴',   '┴',   '┴',   '┴',   '╧',   '╩',   '╧'], _
+            ['TopCross',            '',    '┬',   '┬',   '┬',   '┬',   '╤',   '╦',   '╤',   '─'], _
+            ['HeaderCross',         '',    '┼',   '┼',   '┼',   '┼',   '┼',   '╬',   '┼',   ' '], _
+            ['BottomCross',         '',    '┴',   '┴',   '┴',   '┴',   '╧',   '╩',   '╧',   '─'], _
             _
-            ['TopHorizontal',       '',    '─',   '─',   '─',   '─',   '═',   '═',   '═'], _
-            ['HeaderHorizontal',    '',    '─',   '─',   '─',   '─',   '─',   '═',   '─'], _
-            ['BottomHorizontal',    '',    '─',   '─',   '─',   '─',   '═',   '═',   '═'], _
+            ['TopHorizontal',       '',    '─',   '─',   '─',   '─',   '═',   '═',   '═',   '─'], _
+            ['HeaderHorizontal',    '',    '─',   '─',   '─',   '─',   '─',   '═',   '─',   ' '], _
+            ['BottomHorizontal',    '',    '─',   '─',   '─',   '─',   '═',   '═',   '═',   '─'], _
             _
-            ['HeaderVerticalLeft',  '',    '',    '│',   '│',   '│',   '│',   '║',   '║'], _
-            ['HeaderSeparator',     '',    '│',   '│',   '│',   '│',   '│',   '║',   '│'], _
-            ['HeaderVerticalRight', '',    '',    '│',   '│',   '│',   '│',   '║',   '║'], _
-            ['HeaderLeft',          '',    '',    '├',   '├',   '├',   '├',   '╠',   '╟'], _
-            ['HeaderRight',         '',    '',    '┤',   '┤',   '┤',   '┤',   '╣',   '╢'], _
+            ['HeaderVerticalLeft',  '',    '',    '│',   '│',   '│',   '│',   '║',   '║',   '│'], _
+            ['HeaderSeparator',     '',    '│',   '│',   '│',   '│',   '│',   '║',   '│',   ' '], _
+            ['HeaderVerticalRight', '',    '',    '│',   '│',   '│',   '│',   '║',   '║',   '│'], _
+            ['HeaderLeft',          '',    '',    '├',   '│',   '├',   '├',   '╠',   '╟',   '│'], _
```

Comment:
@WildByDesign
Why do you want to change HeaderLeft and HeaderRight for the default border style 4?

Reply:
I will redo this change in one of my next commits. But no showstopper regarding merging this.